pulsar-client-tools/src/main/java/org/apache/pulsar/admin/cli/CmdBrokers.java

```diff
@@ -64,9 +64,9 @@ void run() throws Exception {
 
     @Parameters(commandDescription = "Update dynamic-serviceConfiguration of broker")
     private class UpdateConfigurationCmd extends CliCommand {
-        @Parameter(names = "--config", description = "service-configuration name", required = true)
+        @Parameter(names = {"-c", "--config"}, description = "service-configuration name", required = true)
         private String configName;
-        @Parameter(names = "--value", description = "service-configuration value", required = true)
+        @Parameter(names = {"-v", "--value"}, description = "service-configuration value", required = true)
```

Comment:
I think we should reserve `-v` for verbose logging. I recognize that we don't have this feature currently, but we could add it, and I don't believe we're using `-v` yet.

Reply:
I found that `-v` is used as a short name for `--value`
https://github.com/apache/pulsar/blob/a43fab0045a93ee864da6bc386bcc0e8bf17bf11/pulsar-client-tools/src/main/java/org/apache/pulsar/admin/cli/CmdNamespaces.java#L2383-L2384
Maybe using `-ve` is also possible If in order to avoid the problem you mentioned.

Reply:
> I found that `-v` is used as a short name for `--value`

Okay, let's leave it as is. I think it's probably unlikely we'll add a -v for verbose logging, and if we do, we can deal with it then.